src/core/types/elasticsearch/index.ts

```diff
@@ -29,4 +29,4 @@ export type ESSearchResponse<
   TOptions extends { restTotalHitsAsInt: boolean } = { restTotalHitsAsInt: false }
 > = InferSearchResponseOf<TDocument, TSearchRequest, TOptions>;
 
-export { InferSearchResponseOf, AggregationResultOf, SearchHit };
+export type { InferSearchResponseOf, AggregationResultOf, SearchHit };
```

Comment:
Can we add someone familiar with the new typings as co-owner of these files? At least for the transition period.

Reply:
will do!